.github/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  CodeQL-Build:
+
```

Comment:
is this file needed?  was it added by accident?

Reply:
removed